R/py_int/covid_run.R

```diff
@@ -1,29 +1,30 @@
-# list.of.packages <- c("rampuaR")
-# new.packages <- list.of.packages[!(list.of.packages %in% installed.packages()[,"Package"])]
 
-# if(length(new.packages)) devtools::install_github("Urban-Analytics/rampuaR", dependencies = F)
+load_rpackages <- function() {
+  list.of.packages <- c("rampuaR")
+  new.packages <- list.of.packages[!(list.of.packages %in% installed.packages()[,"Package"])]
 
-# library(rvcheck)
-# 
-# rampr_version <- check_github("Urban-Analytics/rampuaR")
-# if(!rampr_version$up_to_date) devtools::install_github("Urban-Analytics/rampuaR", dependencies = F)
+  if(length(new.packages)) devtools::install_github("Urban-Analytics/rampuaR", dependencies = F)
```

Comment:
I think the dependencies call here should be `dependencies = c('depends')` to ensure that on a totally fresh setup this ensures all of the rampuaR dependencies are installed, otherwise I think it may lead to an error.

Reply:
Same on line 12